platform/local/flight.go

```diff
@@ -35,6 +36,10 @@ const (
 type LocalFlight struct {
 	destructor.MultiDestructor
 	*platform.BaseFlight
+	Dnsmasq    *Dnsmasq
+	SimpleEtcd *SimpleEtcd
```

Comment:
I'm a bit weary about having a shared `etcd` instance across the entire `Flight`.

One of the RFE's I've had in mind to implement for a while is #799 which could potentially encounter some problems with a singular shared `etcd` instance on some tests (the fleet test comes to mind).

Reply:
Unless I'm missing something, this etcd is only used for the discovery service.